lib/hammer_cli_foreman_remote_execution/options/normalizers.rb

```diff
@@ -1,3 +1,5 @@
+require 'parse-cron'
```

Comment:
No other than minimizing deps on client side, but probably not that important. I'm ok with how is it right now